common/src/main/java/uk/gov/gchq/palisade/User.java

```diff
@@ -64,6 +65,23 @@ public void setField(final String reference, final Object value) {
         Util.setField(this, FIELD_SETTERS, reference, value);
     }
 
+    public Object getUserField(final String key) {
+        requireNonNull(key, "The key cannot be set to null.");
+        return userFields.get(key);
+    }
+
+    public void setUserField(final String key, final Object value) {
+        requireNonNull(key, "The key cannot be set to null.");
+        requireNonNull(key, "The value cannot be set to null.");
+        userFields.put(key, value);
```

Comment:
Please can this be renamed to public User "userField"(...)  and then have a void setUserField(...) which just calls it.